### PR TITLE
Hangman Program

### PR DESCRIPTION
**Pull request by Urban158**
Hangman Program with Gallow Display

### CODE REVIEW

File: HangmanWords.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+program
```
Comment (ADWolfe-LoyNO) [inline HangmanWords.txt:1]: Just grabbed a C++ vocabulary page?? 😉

File: NewmanHangmanwGallow.cpp [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+  string words[SIZE];
+  for (int i = 0; i < SIZE; i++)
+  {
+    inputFile >> word;
+    words[i] = word;
+  }
+  inputFile.close();
+  while (true)
+  {
+    int n = rand() % (25) + 1;
```
Comment (ADWolfe-LoyNO) [inline NewmanHangmanwGallow.cpp:52]: you should use `size` here

File: NewmanHangmanwGallow.cpp [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+      cout << "You have " << MAXCHANCES - miss << "  guesses left." << endl;
+      if (MAXCHANCES == miss)
+      {
+        cout << "You lost the game. The word is: " << word << endl;
+        break;
+      }
+    }
+
+  if (MAXCHANCES == miss)                                                       // Scene 10 is the full display
+  {
```
Comment (ADWolfe-LoyNO) [inline NewmanHangmanwGallow.cpp:122]: you probably want this whole block to be a separate function

File: NewmanHangmanwGallow.cpp [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+    cout <<  "*" << "*" << "*" << "*" << "*" <<"*" << "*" << "*" <<"*" << "*" << "*" << "*" << "*" << "*" << "*" << "*" << endl;
+    cout <<"Sorry incorrect guess, you lose!" << endl;
+  }
+  else if ((MAXCHANCES - miss) == 1)
+  {
+    //symbol = "*";
+
+    cout << "     ***********" << endl;
+    cout << "     *         " << "*" << endl;
+    cout << "     *         " << "*" << endl;
```
Comment (ADWolfe-LoyNO) [inline NewmanHangmanwGallow.cpp:146]: look for a way to combine these repetitions

**Review by ADWolfe-LoyNO — APPROVED** [review]
Good work
This does not have enough comments
Look for ways to reduce duplicate code
Grade: 91 of 100

**Comment by Urban158** [comment]
Yes, I was hoping to have a theme going for this program. 🥺

**Pull request closed without merging**